File: test/fixtures/output/node/native/text-plain.js
const http = require("http");

const options = {
  "method": "POST",
  "hostname": "mockbin.com",
  "port": null,
  "path": "/har",
  "headers": {
    "content-type": "text/plain"
  }
};

const req = http.request(options, function (res) {
  const chunks = [];

  res.on("data", function (chunk) {
    chunks.push(chunk);
  });

  res.on("end", function () {
    const body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.write("Hello World");
req.end();

File: test/fixtures/output/ruby/native/text-plain.rb
require 'uri'
require 'net/http'
require 'openssl'

url = URI("https://httpbin.org/anything")

http = Net::HTTP.new(url.host, url.port)
http.use_ssl = true

request = Net::HTTP::Post.new(url)
request["content-type"] = 'text/plain'
request.body = "Hello World"

response = http.request(request)
puts response.read_body
